README.md

```diff
@@ -19,6 +19,11 @@ npm install //install dependencies
 npm start //start webpack-dev-server with cycle + hot reload
 ```
 
+## Testing
+```
+webpack --config webpack.test.config.js --watch
```

Comment:
can you make this into a new npm task eg: 
`"test-server": "webpack --config webpack.test.config.js --watch"`

then update this to `npm test-server`